Player.cs

```diff
@@ -1,25 +1,51 @@
-﻿using System.Collections.Generic;
-
-namespace DungeonExplorer
-{
-    public class Player
-    {
-        public string Name { get; private set; }
-        public int Health { get; private set; }
-        private List<string> inventory = new List<string>();
-
-        public Player(string name, int health) 
-        {
-            Name = name;
-            Health = health;
-        }
-        public void PickUpItem(string item)
-        {
-
-        }
-        public string InventoryContents()
-        {
-            return string.Join(", ", inventory);
-        }
-    }
+﻿using System.Collections.Generic;
+
+namespace DungeonExplorer
+{
+    public class Player
+    {
+        public string Name { get; private set; }
+        public double Health { get; private set; }
+        public int steps { get; private set; }
+        public double score { get; private set; }
+        private List<string> inventory = new List<string>();
+
+        public Player(string name, int health) 
+        {
+            this.Name = name;
+            this.Health = health;
+        }
+        public void addScore(int points)
+        {
+            score += points;
+        }
+        public void removeScore(int points)
+        {
+            score -= points;
+        }
+        public void pickupItem(string item)
+        {
+            if (item != null)
+            {
+                inventory.Add(item);
+            }
+        }
+        public void potionEffect()
+        {
+            Health += 10;
+        }   

```

Comment:
you could potentially cap health here, for example

```suggestion
        public void potionEffect()
        {
            Health += 10;
            if (Health > 100) Health = 100;
        }   

```

Reply:
I think the player should be able to boost their health if they need to, so I wont change this but thank you for the input.